target_chains/cosmwasm/contracts/pyth/src/contract.rs

```diff
@@ -167,16 +180,30 @@ fn update_price_feeds(
         let batch_attestation = BatchPriceAttestation::deserialize(&data[..])
             .map_err(|_| PythContractError::InvalidUpdatePayload)?;
 
-        let (num_updates, num_new) =
+        let (num_attestations, new_attestations) =
             process_batch_attestation(&mut deps, &env, &batch_attestation)?;
-        total_attestations += num_updates;
-        new_attestations += num_new;
+        num_total_attestations += num_attestations;
+        for new_attestation in new_attestations {
+            total_new_attestations.push(new_attestation.to_owned());
+        }
     }
 
-    Ok(Response::new()
+    let num_total_new_attestations = total_new_attestations.len();
+
+    let response = Response::new();
+
+    #[cfg(feature = "injective")]
+    {
```

Comment:
I think here you can have it in the code:
```
if cfg!(feature = "injective") {
 ...
}
```

Reply:
```
cannot find function `create_relay_pyth_prices_msg` in this scope
not found in this scope
```

I get this error doing it like this